### DISCUSSION TITLE
Suggestion: new Component : cross tabs

### DISCUSSION THREAD

**Discussion by AlexisVS** [squidfunk/mkdocs-material#6680] [Ideas]
I think, it can be good feature to having a cross tabs component

It's pretty usefull for stack a lot of informations inside (visually) a single component.

I have been inspired by [quasar.dev documention](https://quasar.dev/vue-components/table)

|         | --> `tab 1` <--   | tab 2   | tab 3 |
|---------|-------------|---------|-------|
| --> `tab 1_1` <-- | My only one | content | here  |
| tab 1_2 | My only one | content | here  |
| tab 1_3 | My only one | content | here  |

**Comment by alexvoss** [comment]
I am afraid that it is unlikely that this will be taken up as it is a question of markdown processing, which is not what Material for MkDocs does. It relies on [Python Markdown](https://python-markdown.github.io/) and (where configured) on the [PyMdown Extensions](https://facelessuser.github.io/pymdown-extensions/). It makes more sense for new markdown functionality to be implemented there. 

Before you suggest it, though, the maintainers would need a much more precise specification of what you would like to achieve. I am afraid that I could not make sense of the above. Might just be me or perhaps you meant to point to something more specific in the Quasar documentation?

**Reply by AlexisVS** [reply]
Thanks for you reply, PyMdown is a nice extension.
